global-setup.ts

```diff
@@ -1,3 +1,5 @@
+import { setupTestFolder } from './testing/test-setup/src';
+
```

Comment:
`setupTestFolder` is imported but not called, was it an oversight? Or has this import a side effect?

Reply:
removed